tnc/modem.py

```diff
@@ -1251,11 +1251,11 @@ def calculate_fft(self) -> None:
                         range_end = range[1]
                         # define the area, we are detecting busy state
                         #dfft = dfft[120:176] if TNC.low_bandwidth_mode else dfft[65:231]
-                        dfft = dfft[range_start:range_end]
+                        slotdfft = dfft[range_start:range_end]
                         # Check for signals higher than average by checking for "100"
                         # If we have a signal, increment our channel_busy delay counter
                         # so we have a smoother state toggle
-                        if np.sum(dfft[dfft > avg + 15]) >= 400 and not TNC.transmitting:
+                        if np.sum(slotdfft[slotdfft > avg + 15]) >= 400 and not TNC.transmitting:
```

Comment:
@DJ2LS I'm not sure if the >= 400 should/needs to be adjusted since we're looking at a smaller sample size.  Thoughts?

Reply:
This needs to be carefully tested I think. We can reduce this step by step. What about reducing it to 200 as a first test?

Reply:
200 definitely seems better, but may need further refinement.